### DISCUSSION TITLE
Documentation for install a third library like material-ui

### DISCUSSION THREAD

**Discussion by ha-akelius** [strapi/strapi#7037] [Legacy Topics]
I would like to use `material-ui` in Strapi Admin

I didn't find any documentation how to do that :(

could you please recommend me one

**Comment by MattieBelt** [comment]
Can you clarify a bit more where you want to implement material-ui in the admin panel?
Overwrite an existing component or for in your own plugin?

**Comment by ha-akelius** [comment]
I need it to the admin panel,
I am writing my own plugin

**and as you answered me in the slack 👍  **,
I have to install it inside my plugin 


Thank you
